server/src/main/java/org/apache/iotdb/db/utils/TypeInferenceUtils.java

```diff
@@ -102,4 +102,27 @@ public static TSDataType getPredictedDataType(Object value, boolean inferType) {
 
     return TSDataType.TEXT;
   }
+
+  public static TSDataType getAggrDataType(String aggrFuncName, TSDataType dataType) {
```

Comment:
Why we need this new method here?

Reply:
because we need to know the data type of the aggregation result